config/routes.rb

```diff
@@ -50,6 +50,13 @@
     if ForemanTasks.dynflow.required?
       require 'dynflow/web'
       mount ForemanTasks.dynflow.web_console => '/dynflow'
+      if defined? ::Sidekiq
+        require 'sidekiq/web'
+        redis_url = SETTINGS.dig(:dynflow, :redis_url)
+        Sidekiq.redis = { url: redis_url }
+        Sidekiq::Web.set :sessions, false
+        mount Sidekiq::Web => '/sidekiq', :constraints => ForemanTasks::Dynflow::SidekiqConsoleConstraint.new
```

Comment:
If I try to access `/foreman_tasks/sidekiq` as a non-admin user, I get routing error. `403` would be nicer, but in general it prevents unauthorized users to go there. I guess it is ok for now

Reply:
yeah, true, that's what the constraint does.. I will try to improve it - but rather in some followup.